webpack.config.js

```diff
@@ -373,27 +374,42 @@ let appConfig = {
       ),
     },
 
+    fallback: {
+      ...(IS_PRODUCTION
+        ? {
+            vm: require.resolve('vm-browserify'),
+            stream: require.resolve('stream-browserify'),
+            crypto: require.resolve('crypto-browserify'),
+          }
+        : {}),
+    },
+
     modules: ['node_modules'],
     extensions: ['.jsx', '.js', '.json', '.ts', '.tsx', '.less'],
   },
   output: {
     path: distPath,
     filename: '[name].js',
-
-    // Rename global that is used to async load chunks
-    // Avoids 3rd party js from overwriting the default name (webpackJsonp)
-    jsonpFunction: 'sntryWpJsonp',
```

Comment:
Bumped into this the other day and was pretty sure it wasn't used. Good to see it removed

Reply:
It was used fwiw - it was added after Stripe overwrote the `webpackJsonp` global and broke our SPA, but this was removed in webpack5 where it defaults to `package.json -> name`